hyperlinks all the links.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Hyperlinks</title>
+    </head>
+<body>
+<h1 id="top"> This page is about all the links </h1>

```

Comment:
This is an example of an "#top" or id="top"... so an anchor lower on the page (line 41) can say <a href="#top">Go to TOP</a> and it'll send the cursor to this h1.

Reply:
so line 41 should be <a href="go to top"> or <a href="#top"> Go to TOP</a>?